README.md

```diff
@@ -33,12 +50,16 @@ $ brew install gnupg.rb
 On Linux (and also on OS X if you don't want to use Homebrew):
 
 ```
-$ ./install_locally.sh
+$ ./build_gpg/install_locally.sh
 ```
 
 Of course, you should never run arbitrary executable code from the Internet.
 You should verify that the scripts do what they are expected to do (they're
-pretty short).
+pretty short). A dummy RSA key will be generated according to the parameters in the **keyparams** file and added to the keyring. Do not forget to remove it after usage:
+
+```
+$gpg --delete-secret-and-public-keys Flush
```

Comment:
nit: `$gpg` -> `$ gpg` 

I want to make sure that this doesn't get misinterpreted as an environment variable